lega/conf/defaults.ini

```diff
@@ -34,15 +34,10 @@ s3_chunk_size = 4194304
 
 ## Connecting to Local Broker
 [broker]
+connection = amqps://guest:guest@mq:5672/%2F?heartbeat=0&connection_attempts=30&retry_delay=10
 enable_ssl = no
```

Comment:
`enable_ssl` is not used any more since we default to ssl only.

Reply:
done.